sources/Context.cpp

```diff
@@ -51,4 +61,23 @@ Context::process() {
   LOG(info) << MESS(SlackNode, slackNode_->id, static_cast<unsigned int>(slackNodeOrigin_)) << LOG_ENDL;
 }
 
+void
+Context::exportOutputs() {
+  LOG(info) << MESS(ExportInfo, basename_) << LOG_ENDL;
+
+  // Job
+  jobWriter_ = std::unique_ptr<outputs::Job>(new outputs::Job(outputs::Job::JobDefinition(config_.outputDir(), basename_, dynawLogLevel_)));
```

Comment:
not sure why we write a job when the input of the Simulation class in Dynawo is directly a jobEntry

Reply:
> 2 main things here:
> 
>     * I don't think we need to write a job on the disk, since we can call a dynawo simulation directly with a jobEntry
> 
>     * User shouldn't have to configure anything related to constant files

For a first draft, we create the input files to be able to test that we provide the correct informations to dynawo siulation

Reply:
This can be tested directly on the jobEntry (and displayed if needed). for others files we can have a first shot with files written on the disk, but for the jobs it seems useless

Reply:
(sorry if I wasn't clear on this topic)

Reply:
For me, we have to perform the same tests for all files, even if it is less relevant for the job file. That way all inputs of dynawo simulation are tested the same way in draft. JOB entry generation will be done when the simulation will be launched from inside the launcher

Reply:
I don't think we need a step in which dynawo is called as a blackbox since we already know this works.
@marcochiaramello @cossartque what do you think? I strongly think that we can already go to a version in which the simulation is directly launched from inside the tool as this is something that was already tested extensively.

Reply:
I agree, I also think that we can launch the simulation directly from inside the tool.

Reply:
agreed

Reply:
OK, I'll reverse this in another pull request

Reply:
OK thanks! and sorry for the misunderstanding